packages/workspace/src/workspace/nacl_files/nacl_files_source.ts

```diff
@@ -308,7 +314,8 @@ const buildNaclFilesSource = (
 
   const getSourceMap = async (filename: string): Promise<SourceMap> => {
     const parsedNaclFile = (await getState()).parsedNaclFiles[filename]
-    const key = cacheResultKey(parsedNaclFile.filename, parsedNaclFile.timestamp)
+    const key = cacheResultKey({ filename: parsedNaclFile.filename,
+      timestamp: parsedNaclFile.timestamp })
```

Comment:
nit: applies to other places as well
```suggestion
    const key = cacheResultKey(parsedNaclFile)
```